tests/database/database_test.cpp

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024s EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gtest/gtest.h>
+
+#include "database/database.hpp"
+
+using namespace testing;
+
+/***********************************************************************************************************************
+ * Static
+ **********************************************************************************************************************/
+
+class DatabaseTest : public Test {
+protected:
+    void TearDown() override { std::remove(mFileName.c_str()); }
+
+    const aos::Array<uint8_t> StringToDN(const char* str)
+    {
+        return aos::Array<uint8_t>(reinterpret_cast<const uint8_t*>(str), strlen(str) + 1);
+    }
+
+protected:
+    std::string mFileName = "test.db";
+    Database    mDB;
+};
+
+/***********************************************************************************************************************
+ * Tests
+ **********************************************************************************************************************/
+
+TEST_F(DatabaseTest, Init)
```

Comment:
Useless test. Init is tested in other test cases.

Reply:
Remove